recipes/cpython/all/patches/3.10.3-0005-do-not-search-for-system-tkinter.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.py
++++ setup.py
+@@ -2106,6 +2106,8 @@
+         if self.detect_tkinter_fromenv():
+             return True
+ 
++        return False
++
+         # Rather than complicate the code below, detecting and building
```

Comment:
Please modify this patch to keep the number of lines the same.
So remove an empty line and replace it with the `return False`.
Keeping the number of lines the same makes it easer to stack and re-order patches: the line offset remains the same.